prov/gni/src/gnix_ep.c

```diff
@@ -501,6 +881,7 @@ static int gnix_ep_control(fid_t fid, int command, void *arg)
 		if (ep->type == FI_EP_RDM) {
 			dom = ep->domain;
 			for (i = 0; i < dom->fabric->n_wc_dgrams; i++) {
+				assert(ep->recv_cq != NULL);
```

Comment:
Is this temporary or are we going to require an RX CQ to be bound to each RDM EP?